frontend/app/src/types/history/events/index.ts

```diff
@@ -87,7 +87,9 @@ export const HistoryEvent = z.object({
   notes: z.string().nullish(),
   counterparty: z.string().nullish(),
   product: z.string().nullish(),
-  address: z.string().nullish()
+  address: z.string().nullish(),
+  isExit: z.boolean().nullish(),
+  validatorIndex: z.number().nullish()
```

Comment:
For this as soon as there is a way to distinguish between the types let's separate them. Also, we need to make sure that we don't use `nullish`. In most cases, `nullish` is overly broad, since the API is either using one or the other. The correct approach should be either `nullable` or `optional`.